packages/data-point/lib/reducer-entity/factory.js

```diff
@@ -19,7 +19,10 @@ function ReducerEntity () {
 module.exports.ReducerEntity = ReducerEntity
 
 function isEntity (source) {
-  return source.match(/^[^$]([\w.]+):?([\w.]+)(\[])?/) !== null
+  return (
+    typeof source === 'string' &&
```

Comment:
hmm why do you use `typeof` here but `_.isString` in `reducer-function/factory.js`?

Reply:
this mostly I didnt have the dependency on the file, so, why add it just for that test, lazy me, I'll add it thanks!